src/packages/admin-ui-components/src/table/component.tsx

```diff
@@ -38,10 +38,8 @@ const columnsForEntity = <T extends TableRowItem>(
 			? ({ row }: FormatterProps<T, unknown>) => {
 					// Without stopping propagation on our links, the grid will be notified about the click,
 					// which is not what we want. We want to navigate and not let the grid handle it
-					const gobbleEvent = useCallback(
-						(e: MouseEvent<HTMLAnchorElement>) => e.stopPropagation(),
-						[]
-					);
```

Comment:
Instead of removing the `useCallback` optimisation, can we just move it out of the `map` so there's one instance for all the rows, e.g. put this on line 28?

Reply:
Thanks @thekevinbrown, I've moved it out completely to a single, global function.